aea/cli/core.py

```diff
@@ -193,6 +197,100 @@ def add_key(ctx: Context, type_, file):
     )
 
 
+@cli.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice(
+        [DefaultCrypto.identifier, FetchAICrypto.identifier, EthereumCrypto.identifier]
+    ),
+    required=True,
+)
+@pass_ctx
+def get_address(ctx: Context, type_):
+    """Get the address associated with the private key."""
+    try_to_load_agent_config(ctx)
+
+    _verify_or_create_private_keys(ctx)
+    private_key_paths = dict(
+        [
+            (identifier, config.path)
+            for identifier, config in ctx.agent_config.private_key_paths.read_all()
+        ]
+    )
+    try:
+        wallet = Wallet(private_key_paths)
+        address = wallet.addresses[type_]
+        logger.info("{}".format(address))
```

Comment:
I think we should just use `print`, because this will prepend `info: `

![Screenshot from 2020-01-25 12-07-02](https://user-images.githubusercontent.com/9467617/73120193-6e221780-3f6b-11ea-82b0-3b90b297c883.png)